### DISCUSSION TITLE
Example of Vuejs app with vitejs

### DISCUSSION THREAD

**Discussion by BhaskaranR** [nxext/nx-extensions#372] [Q&A]
Excellent work so far trying to integrate nx with vitejs. do you have a working example of creating a vitejs app with vuejs?  how do i extend the vite js config, how to i import a vite library to vite app?

**Comment by DominikPieper** [comment]
Hey @BhaskaranR ,
we didn’t try any framework than solid, svelte and react yet. It should be easy to ad the vue vite starter by generating a generic vite app and copy once the vue files, install dependencies and configure the vite.config.js. We don’t have any generators for vue so that’s something you’d need to do by hand
